it-tests/BasicFlow.test.ts

```diff
@@ -0,0 +1,51 @@
+import { By, CustomEditor, EditorView,  until,  VSBrowser, WebDriver, WebView } from 'vscode-extension-tester';
+import { assert } from 'chai';
+import * as path from 'path';
+
+describe('Kaoto basic development flow', function () {
+	this.timeout(120000);
+
+	const workspaceFolder = path.join(__dirname, '../test Fixture with speci@l chars');
+
+	let driver: WebDriver;
+
+	before(async function() {
+		this.timeout(20000);
+		driver = VSBrowser.instance.driver;
+		await VSBrowser.instance.openResources(workspaceFolder);
+	});
+
+	after(async function() {
+		const editorView = new EditorView();
+		await editorView.closeAllEditors();
+	});
+
+	it('Open "empty.kaoto.yaml" file and check Kaoto UI is loading', async function () {
+		this.timeout(60000);
```

Comment:
Does the docker image need to be downloaded for each assertion? Or why are we having so many different timeouts?

Reply:
Need to be downloaded only one time.

This timeout is globally for the test execution.
We need to specify timeout for some specific driver.wait as default timeout for some is at 5s (if i remember well). The sum of all timeout in driver.wait should be lower than the global one.